.github/workflows/coverity.yml

```diff
@@ -1,15 +1,40 @@
 name: Coverity scan
 on:
-  # run once daily at 00:30 UTC due to
+  # coverity_daily job runs once daily at 00:30 UTC due to
   # https://scan.coverity.com/faq#frequency
   schedule:
     - cron: "30 0 * * *"
+  # coverity_label job triggers if PR has 'coverity' label set.
+  # This job likely cannot be re-run on the same day or we will
+  # hit the build submission limit (see above link)
+  pull_request_target:
+    branches: [master]
```

Comment:
shouldn't you narrow it down to
```
types:
- labeled
```

Reply:
It could be, do you suggest if someone wants to run covscan against their PR more than once they should just remove and re-add the label? If yes I will add a comment in this workflow about this.

Reply:
Yes. I think that current implementation will run covscan on each push if label is set, won't t? We need to trigger it only on desire. So either by setting the label or  maybe by `/covscan` comment.

Reply:
Okay makes sense, I added the 

```
types:
- labeled
```